commit ce2ae775a5e1a6753f3a5aff17580d5d05f6a38f
Author: Mirko Parthey <mirko.parthey@web.de>
Date:   Wed Feb 15 23:31:30 2017 +0100

    MIPS: BCM47XX: Fix button inversion for Asus WL-500W
    
    commit bdfdaf1a016ef09cb941f2edad485a713510b8d5 upstream.
    
    The Asus WL-500W buttons are active high, but the software treats them
    as active low. Fix the inverted logic.
    
    Fixes: 3be972556fa1 ("MIPS: BCM47XX: Import buttons database from OpenWrt")
    Signed-off-by: Mirko Parthey <mirko.parthey@web.de>
    Acked-by: Rafał Miłecki <rafal@milecki.pl>
    Cc: Hauke Mehrtens <hauke@hauke-m.de>
    Cc: linux-mips@linux-mips.org
    Patchwork: https://patchwork.linux-mips.org/patch/15295/
    Signed-off-by: James Hogan <james.hogan@imgtec.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1211_ce2ae775a5e1a6753f3a5aff17580d5d05f6a38f.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1211_ce2ae775a5e1a6753f3a5aff17580d5d05f6a38f.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
